git-tar/function/ops.go

```diff
@@ -306,109 +308,128 @@ func clone(pushEvent sdk.PushEvent) (string, error) {
 
 func deploy(tars []tarEntry, pushEvent sdk.PushEvent, stack *stack.Services, status *sdk.Status, payloadSecret string) error {
 
-	failedFunctions := []string{}
-	owner := pushEvent.Repository.Owner.Login
-	repoName := pushEvent.Repository.Name
-	url := pushEvent.Repository.CloneURL
-	afterCommitID := pushEvent.AfterCommitID
-	installationID := pushEvent.Installation.ID
-	sourceManagement := pushEvent.SCM
-	privateRepo := pushEvent.Repository.Private
-	repositoryURL := pushEvent.Repository.RepositoryURL
-
-	c := http.Client{}
-	gatewayURL := os.Getenv("gateway_url")
+	workCh := make(chan int, 4)
```

Comment:
`4` should be a const and also let's set it to `2` so we don't cripple the system.